Tribler/Test/Core/test_session.py

```diff
@@ -89,6 +89,7 @@ def test_add_torrent_def_to_channel(self):
         """
         Test whether adding a torrent def to a channel works
         """
+        self.session.notifier.use_pool = False
```

Comment:
what race condition?
Also is disabling the thread pool usage for this test the way to go?